README.md

```diff
@@ -1,64 +1,73 @@
-# oam-kubernetes-implementation
+# OAM Controllers
+
+This repo contains Kubernetes controllers for [core](https://github.com/oam-dev/spec/blob/master/3.workload.md#core-workload) and [standard](https://github.com/oam-dev/spec/blob/master/3.workload.md#standard-workload) OAM workloads/traits/scopes. 
+
+The main target audience of this repository is OAM workloads/traits/scopes developers. If you are looking for end user experience of OAM application on Crossplane, you probably want to check its [getting started doc](https://crossplane.io/docs/v0.11/getting-started/install-configure.html) instead.
 
-This OAM Kubernetes implementation allows an application developer/operator to compose a cloud-native application following the
- OAM v1alpha2 spec and run it on a kubernetes cluster.  
 
 ## Prerequisites
 
 1. go version 1.12+
-2. Kubernetes version v1.17+ with KubeCtl configured
+2. Kubernetes version v1.16+ with kubectl configured
 3. Helm 3.0+ 
+4. [Crossplane](https://github.com/crossplane/crossplane) is installed as OAM Kubernetes runtime
 
 
-## Getting started
+## (Optional) In case you don't know how to install Crossplane
 
-The functionality of this addon can be demonstrated with the following steps:
-
-* Install cert manager
+Here's a minimal version:
+```console
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
 ```
-kubectl create namespace cert-manager
 
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.14.0/cert-manager.yaml
+Please feel free to ignore this warning, which is cause by helm v2 compatibility issue: 
 ```
-More detailed instructions for cert manager can be found in the [Cert-manager docs](https://cert-manager.io/docs/installation/kubernetes/).
-
-* Install OAM Application Controller
+manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
 ```
-kubectl create namespace crossplane-system
 
-helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+## Install OAM controllers
 
-helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
-```
+#### Clone repo
 
-More detailed instructions can be found in the [Crossplane docs]( https://crossplane.io/docs/v0.8/install-crossplane.html).
-
-* Install OAM Core workload and trait controllers
+```console
+git clone git@github.com:crossplane/addon-oam-kubernetes-local.git
+cd ./addon-oam-kubernetes-local
+```
 
+#### Install cert manager
```

Comment:
Not sure if we still need this part.

Reply:
I think we have to remove webhook along with cert manager if we want to change K8s version requirement to 1.16

Reply:
webhook is already disabled, so we can remove this part.

Reply:
I tested on minikube and it works with K9s version 1.16.2+, it does not work with 1.16.0 or 1.16.1 due to errors on Crossplane installation on those versions.